src/util.cpp

```diff
@@ -75,11 +75,21 @@ float round_to_float(double v) {
     return roundf(v * powl(10, 6)) / powl(10, 6);
 }
 
+struct ggml_tensor * reciprocal(ggml_context * ctx, struct ggml_tensor * x) {
+    TTS_ASSERT(x->ne[0] == 1);
+    static constexpr float one = 1.0f;
+    ggml_tensor * numerator = ggml_new_tensor_2d(ctx, GGML_TYPE_F32, 1, x->ne[1]);
+    // stride hack so that the scalar numerator can be divided by x;
```

Comment:
stride ***tricks*** refers to https://numpy.org/devdocs/reference/generated/numpy.lib.stride_tricks.html